osm/src/main/resources/org/orbisgis/geoclimate/osm/urbanAreasParams.json

```diff
@@ -45,36 +59,42 @@
         "residential"
       ]
     },
-    "heavy_industry": {
-      "industrial": [
-        "refinery"
+    "industrial": {
+      "landuse": [
+        "industrial"
       ]
     },
-    "light_industry": {
+    "port": {
       "landuse": [
-        "industrial", "port"
+        "port"
+      ]
+    },
+    "industrial": {
+      "industrial": [
+        "refinery"
       ]
     },
     "government": {
       "landuse": [
         "institutional"
       ]
     },
-    "social_building": {
-    "amenity": [
-      "community_centre"
-    ]},
+    "community_centre": {
+      "amenity": [
+        "community_centre"
+      ]
+    },
     "military": {
       "landuse": [
         "military"
       ]
     },
-    "transport": {
```

Comment:
This will introduce some issue in urban areas because this type will not be pushed when the buildings are formatted. Rigth ?